spec/requests/included_resources_spec.rb

```diff
@@ -58,11 +56,11 @@
 
         it { is_expected.to be_successful }
 
-        let(:comments_policy_scope) { Comment.limit(1) }
-
-        it 'includes only comments allowed by policy scope' do
-          expect(json_included.length).to eq(1)
-          expect(json_included.first["id"]).to eq(comments_policy_scope.first.id.to_s)
+        it 'includes only comments allowed by policy scope and associated with the article' do
+          expect(json_included.length).to eq(article.comments.count)
```

Comment:
Hmm so does this now test that there are more records than were allowed? I.e. do we lose test coverage on the pundit policy scope being applied?

Reply:
I moved the specs specifically testing for limited scopes to https://github.com/venuu/jsonapi-authorization/pull/119/files#diff-f3a557fdd906ce389d7b5d190c721b99R220. I had to move them out of this shared spec since the behavior is different for GET (which returns the relationships but limited by the scope) vs POST (which should return a 404).

Reply:
Nice, thanks!